src/CommunityToolkit.Maui/Alerts/Toast/Toast.android.cs

```diff
@@ -25,7 +25,7 @@ public partial class Toast
 		var styledText = new SpannableStringBuilder(Text);
 		styledText.SetSpan(new AbsoluteSizeSpan((int)TextSize, true), 0, Text.Length, 0);
 
-		NativeToast = Android.Widget.Toast.MakeText(Platform.AppContext, styledText, GetToastLength(Duration))
+		NativeToast = Android.Widget.Toast.MakeText(Platform.CurrentActivity.Window?.DecorView.FindViewById(Android.Resource.Id.Content)?.RootView?.Context, styledText, GetToastLength(Duration))
```

Comment:
Why can't we use the `AppContext`?

Can we avoid the `Native` word? Maybe we can use `PlatformToast`

Reply:
Because we need a context of the RootView, while the AppContext is Application context